src/main/java/com/opentable/server/ServerLoggingConfiguration.java

```diff
@@ -38,7 +42,12 @@
     public void onApplicationEvent(ApplicationEvent event) {
         if (event instanceof ApplicationEnvironmentPreparedEvent) {
             ApplicationEnvironmentPreparedEvent castEvent = (ApplicationEnvironmentPreparedEvent) event;
-            String serviceName = castEvent.getSpringApplication().getMainApplicationClass().getPackage().getImplementationTitle();
+            // Prefer OT, then Dmitry, then Manifest
```

Comment:
Why not use the service name in the service info bean?

Reply:
You can't get it this early in lifecycle unfortunately. Remember the logger needs to be set up before the context is built, or we dont get bad wiring logging